less/sidemenu.less

```diff
@@ -80,14 +80,33 @@ body.menu-toggled #content-toggled {
     }
   }
 
-  @media (max-height: 745px) {
+  /********************************************************
+  fine compute size for ad display toggler
+  if place is missing, hide social links before hiding ads.
+
+  WHAT            SIZE  CALCUL
+  Logo            80
+  Navigation      185   5*(1+8+20+8)
+  Margin          15
+  Advertising     320
+  Margin          15
+  Links           30
+  Margin          15
+  Social networks 35
+  Margin          15
+
+  Total sans SN   660    35+15
+  Total All       710
+  ********************************************************/
+
+  @media (max-height: 660px) {
```

Comment:
you could use variables for size constants, it would be more explicit

Reply:
done